taxi/models.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+
```

Comment:
Pick up one approach, either leaving one blank line between the class declaration and fields or not. You should be consistent throughout your code. I would suggest do not use one blank line as it is done for the `Car` class.